src/components/Portfolio/LoanList.vue

```diff
@@ -0,0 +1,234 @@
+<template>
+	<div class="loan-list-container tw-mt-4">
+		<div class="loan-list-scroll-container tw-relative">
+			<div class="loan-list-inner-container tw-overflow-x-auto tw-min-w-full">
```

Comment:
Are these classes `loan-list-container`, `loan-list-scroll-container`, and `loan-list-inner-container` being used?

Reply:
Not anymore, removed.